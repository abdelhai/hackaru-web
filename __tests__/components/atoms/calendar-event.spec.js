import { addDays } from 'date-fns';
import Factory from '@/__tests__/__setups__/factory';
import CalendarEvent from '@/components/atoms/calendar-event';

describe('CalendarEvent', () => {
  it('render correctly', () => {
    const wrapper = new Factory(CalendarEvent, {
      propsData: {
        color: '#ff0000',
        title: 'Review',
        startedAt: `${new Date()}`,
        stoppedAt: `${addDays(new Date(), 1)}`
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
