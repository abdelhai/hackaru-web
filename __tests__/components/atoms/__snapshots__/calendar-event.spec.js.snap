// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarEvent render correctly 1`] = `
<article
  style="border-color: #ff0000; background-color: rgba(255, 0, 0, 0.33399999999999996);"
>
  <h1>
    Review
  </h1>
   
  <p>
    1 day
  </p>
</article>
`;
